docs/services/data/_index.md

```diff
@@ -103,6 +103,18 @@ To configure cloud sync, see [configure cloud sync](../data/configure-cloud-sync
 
   {{< /alert >}}
 
+## Data query
+
+Once you have synced data to the cloud, you can query certain types of data directly in the cloud using MQL, the [MongoDB query language](https://www.mongodb.com/docs/manual/tutorial/query-documents/).
```

Comment:
'certain types of data' => 'tabular data'

I would specify that you can query it in MQL or SQL.